CMSgov/design-system
# [WNMGDS-487] Add aria-label to FormLabel label and ChoiceList legend

### Added
- Added `ariaLabel` prop `<FormLabel>` `<label>` element

Closes #708


## How to test `<FormLabel>`
1. Run the doc site locally 
1. add the `aria-label` prop to the `<FormLabel>` example 
1. See that `aria-label` is rendered on the `<label>` element of the `<FormLabel>`  example

## How to test aria-label text on legend
1. Run the doc site locally 
1. Add the following code to the <ChoiceList> example to render the `arial-label` on the legend for the choice 
```JSX
<fieldset>
       <FormLabel component="legend" aria-label="testing Aria">
           Legend text   
       </FormLabel>
       <Choice defaultChecked hint="Checkbox A hint" name="checkbox_choice" type="checkbox" value="a">
           Checkbox A
       </Choice>
       <Choice name="checkbox_choice" type="checkbox" value="b">
           Checkbox B
       </Choice>
</fieldset>
```
1. See that `aria-label` is rendered on the `<legend>` element of the `<ChoiceList>` example
